aemedge/blocks/channel-lookup/channel-lookup.css

```diff
@@ -132,6 +129,10 @@ input[type="number"] {
     -moz-appearance: textfield;
 }
 
+picture {
+    display: inline!important;
```

Comment:
Every class in this CSS file needs to be prefixed by ".channel-lookup", otherwise you are affecting all pictures, all sections on whatever page this is on.
For this example, you also don't need "!important" since you will be targeting it specifically in the class.

```
.channel-lookup .picture {
display: inline;
}
```